run.py

```diff
@@ -932,12 +932,23 @@ def collect_recipe(ceph_cluster):
     """
     version_datails = {}
     client_node = ceph_cluster.get_ceph_objects("client")
+
     out, rc = client_node[0].exec_command(
         sudo=True, cmd="podman --version | awk {'print $3'}"
     )
     output = out.read().decode().rstrip()
-    log.info(f"Podman Version {output}")
-    version_datails["PODMAN"] = output
+    if output:
+        log.info(f"Podman Version {output}")
+        version_datails["PODMAN"] = output
+
+    out, rc = client_node[0].exec_command(
```

Comment:
Add `check_ec=False` as it would raise an exception... the environment could have either of them.